plugins/ibm_qradar/plugin.spec.yaml

```diff
@@ -0,0 +1,982 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: ibm_qradar
+title: IBM QRadar
+vendor: rapid7-sacumen
+support: community
+status: []
+supported_versions: [7.3.3]
+description: IBM QRadar reduces billions of events and flows into a manageable number of actionable offenses that are prioritized by their impact on your business operations. This plugin allows you to use IBM QRadar to orchestrate and automate Ariel search queries and automate offense management
+version: 1.0.0
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/plugins/ibm_qradar
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://www.ibm.com
+tags:
+  - qradar
+  - ariel
+  - aql
+  - offenses
+  - orchestration
+hub_tags:
+  use_cases: [asset_inventory, vulnerability_management, offensive_security]
+  keywords: [offense, ariel, search]
```

Comment:
The Unapproved Keywords validator flags `offense` and `ariel` as unsupported. This is not a big deal and merging this PR would simply introduce them, but to satisfy the validator we can use these tags instead. This suggestion is just a recommendation and is not required to merge.
```suggestion
  keywords: [siem, search]
```